spec.bs

```diff
@@ -378,7 +440,7 @@ Moreover, each {{SharedStorageWorklet}}'s [=global scopes|list of global scopes=
         1. If [=this=]'s [=SharedStorageWorklet/has cross-origin data origin=] is false, return a [=promise rejected=] with a {{TypeError}}.
     1. If |options|["`resolveToConfig`"] is true, [=resolve=] |resultPromise| with |pendingConfig|.
     1. Otherwise, [=resolve=] |resultPromise| with |urn|.
-    1. Let |indexPromise| be the result of running [=get the select-url result index=], given [=this=], |name|, |urlList|, and |options|.
+    1. Let |indexPromise| be the result of running [=get the select-url result index=], given [=this=], |name|, |urlList|, |workletDataOrigin| and |options|.
     1. [=Upon fulfillment=] of |indexPromise| with |resultIndex|, perform the following steps:
```

Comment:
I don't see  |workletDataOrigin|  specified earlier in the steps for the selectURL algo.  Should we add a step to set it?

Reply:
I think this is set in step 14 of the [algorithm](https://pr-preview.s3.amazonaws.com/WICG/shared-storage/pull/179.html#dom-sharedstorageworklet-selecturl) :)

Reply:
Thanks, missed that!  I thought I had fully expanded the algo before I did a search, but I hadn't, grrr.